pyproject.toml

```diff
@@ -21,22 +21,37 @@ classifiers = [
 requires-python = ">=3.10"
 license = { "text" = "Apache Software License" }
 dependencies = [
```

Comment:
how about using `==` for stable installation?

Reply:
Thank you, this is a good idea, especially for packages like formatting tools or other libraries that don’t get frequent updates. 
However, there are some libraries, like `datasets`, that are very actively developed (new version in almost one month). For these, using the specific version might prevent us from benefiting from important bug fixes or new features in a timely manner.

So how about using `==` for more stable or rarely updated packages, and using a more flexible version specifier (like `>=`) for actively maintained ones?

https://pypi.org/project/datasets/#history